File: modulo1/segmento3/apunte1/ejercicio5.web/ejercicio5.js
function testGeneroEdad(){
	const sex = prompt('Ingrese su genero: ');
	const age = parseInt(prompt('Ingrese su edad: '));

	if(sex == 'Masculino')
		alert('Usted es hombre');
	
	if(age >= 18)
		alert('Usted es mayor de edad');
}

File: modulo1/segmento3/apunte1/Ejercicio5.java
import java.util.Scanner;

public class Ejercicio5 {
	public static void main(String args[]){
		Scanner teclado = new Scanner(System.in);

		System.out.print("Genero: ");
		String sex = teclado.nextLine();

		System.out.print("Edad: ");
		Integer age = Integer.parseInt(teclado.nextLine());

		if(sex.equals("Masculino"))
			System.out.println("Usted es hombre");

		if(age >= 18)
			System.out.println("Usted es mayor de edad");
	}
}
